homeassistant/components/packages.py

```diff
@@ -0,0 +1,76 @@
+"""
+Support for Packages.
+
+For more details about this component, please refer to the documentation at
+https://home-assistant.io/components/packages/
+"""
+import logging
+
+import voluptuous as vol
+
+import homeassistant.helpers.config_validation as cv
+
+_LOGGER = logging.getLogger(__name__)
+
+DOMAIN = 'packages'
+CONF_PACKAGES = DOMAIN
+
+_COMPONENT_SCHEMA = vol.Schema({cv.slug: vol.Any(dict, list)})
```

Comment:
This is the only supported structure by packges. Also refer to longer comment about `switch 2` etc